src/components/RightPanel.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Flex } from "@chakra-ui/react";
+import ProfileLinks from "./ProfileLinks";
+import ColorModeSwitch from "./ColorModeSwitch";
+
+function RightPanel() {
+  return (
+    <Flex
+      id="right-panel-container"
+      direction={"column"}
+      justifyContent={"space-between"}
+      alignItems={"center"}
+      h={"100%"}
+      py={2}
+    >
+      <Box id="toggle-theme-button">
+        <ColorModeSwitch />
+      </Box>
+      <ProfileLinks />
```

Comment:
I think if we are following the convention to assign Id's to every element then we should do the same here as well.

Reply:
Sure. Thanks for the feedback. I will push an update with an ID assigned to every element.